src/charts/tooltip.js

```diff
@@ -527,7 +525,17 @@ define(function(require){
                     line.push(word);
                     tspan.text(line.join(' '));
 
-                    if (tspan.node().getComputedTextLength() > width) {
+                    // fixes for IE wrap text issue
+                    let temp;
+                    temp = document.createElementNS("http://www.w3.org/2000/svg","text");
+                    temp.appendChild(document.createTextNode(line.join(' ')));
+                    temp.setAttribute('id', 'tempnode');
+                    temp.setAttribute("x",38);
+                    temp.setAttribute("y",18);
+                    document.getElementsByTagName("svg")[0].appendChild(temp);
+                    const textWidth = document.getElementsByTagName("svg")[0].lastChild.getBBox().width;
```

Comment:
I understand we do this to measure the text, right?

We have a small function in our text helper, `getTextWidth`, would this function work?

Reply:
Yes, its to measure the rendered width.  I think the issue is with IE not knowing the size of anything until it's in the DOM.  this link https://stackoverflow.com/a/25168951 shows how to get the correct size in internet explorer

Reply:
I see, I wonder if our method would work in IE11, as it is based in canvas

Reply:
I just changed that to 

```
const textWidth = getTextWidth(line.join(' '), 16, 'Karla, sans-serif');
```

and it does provide the exact same functionality.  You have to pass the font size and font family into the function and it'll work.

Reply:
Awesome!
That function has a default font size and font family, but they are different. Maybe we should update those?

Happy that this worked!